crates/solvers/src/domain/auction.rs

```diff
@@ -46,5 +64,5 @@ impl Price {
 
 /// The estimated effective gas price that will likely be used for executing the
 /// settlement transaction.
-#[derive(Clone, Copy, Debug)]
+#[derive(Clone, Copy, Debug, Default)]
```

Comment:
nit: I would undo this change. `default` gas price seems to be neede for unit test convenience, but I don't think it conceptually makes sense (is the default 0? this won't work with most chain, not even test ones with default parameters).

Reply:
https://github.com/cowprotocol/services/pull/1513/commits/32cc414ac5be918d476da86ee75ebf9b4ce1a63a